frontend/main.py

```diff
@@ -0,0 +1,50 @@
+"""Frontend CLI for Birdman"""
+
+import click
+
+@click.command()
+def details():
+
+    """Function for details to be written in txt file."""
+
+    venue_decided = click.confirm("Venue Decided")
+
+    if venue_decided:
+
+        meetup_link = click.prompt("Meetup Link", type=click.STRING)
+        poster_link = click.prompt("Poster Link", type=click.STRING)
+        tg_usrname = click.prompt("Telegram Username", type=click.STRING)
+        talk_decided = click.confirm("Talks Decided")
+
+        if talk_decided:
+            click.echo("Processing...(saving to .birdman/<date>/meetup.txt)\n")
+            click.echo("Join us for the next ILUG-D meetup at <Date/Time> at <Venue>\n")
```

Comment:
dont print code. actually save the template to .birdman/<date>/meetup.txt first and then print it out

Reply:
what template are you talking about here?

Reply:
Join us for the next ILUG-D meetup at <Date>.....  
Use fstrings